client/zweb/renderer/index.js

```diff
@@ -82,7 +81,7 @@ class Renderer {
             },
             decrypt_data: async function (encryptedData, unparsedEncryptedSymmetricObjJSON) {
                 const host = this.host;
-                const {privateKey} = this.renderer.ctx.wallet.config;
+                const privateKey = this.renderer.ctx.wallet.getNetworkAccountPrivateKey();
```

Comment:
I'm not sure, but I think the wallet should not provide the private key, just services signing with it.

Reply:
@brianunlam, you're right, but we cannot avoid it completely ATM. In some cases we still need it. Not for decryption in particular, but there are two more places where we still need to get the private key. We'll think how to improve it soon. I already have some ideas.